pyteal/ast/subroutine.py

```diff
@@ -320,6 +490,12 @@ def __call__(self, fnImplementation: Callable[..., Expr]) -> SubroutineFnWrapper
 Subroutine.__module__ = "pyteal"
 
 
+def routerSubroutine(fnImplementation: Callable[..., Expr]) -> SubroutineFnWrapper:
```

Comment:
```suggestion
def router_subroutine(fnImplementation: Callable[..., Expr]) -> SubroutineFnWrapper:
```

Making this suggestion as going forward we are trying to be more pythonic in our naming conventions. Of course, this begs the question - what do we do about the fact that we are using different snake conventions in the same module?

Reply:
👍